### PR TITLE
Add the posibility to change multible config files.

### PR DESCRIPTION
**Pull request by jochumdev**
This pull requests adds the posibility to manage multible config files and fills the pillar.example.sls.

### CODE REVIEW

File: aptcacherng/defaults.yaml [hunk 1]
```diff
@@ -4,7 +4,8 @@ Debian:
     - apt-cacher-ng
   service: {}
   config:
-    manage:
-      - acng_conf
-    acng_conf: {}
```
Comment (arnisoph) [inline aptcacherng/defaults.yaml:9]: the underscore was used to access a dict. the dot in the name break this. :)

File: aptcacherng/init.sls [hunk 2]
```diff
@@ -14,16 +14,13 @@ aptcacherng:
...
-    - template: jinja
-    - watch_in:
-      - service: aptcacherng
-{% endif %}
+     - managed
+     - name: {{ datamap.conf_dir|default('/etc/apt-cacher-ng') }}/{{ k }}
+     - mode: 644
+     - user: root
+     - group: root
+     - contents_pillar: aptcacherng:configs:{{ k }}:content
```
Comment (arnisoph) [inline aptcacherng/init.sls:25]: `watch_in` is missing

**Comment by arnisoph** [comment]
@pcdummy thank you for the patch!

I'm going to merge this, but there are some small changes needed to keep expected behaviour.

File: aptcacherng/defaults.yaml (hunk 1 above)
Comment (jochumdev) [inline aptcacherng/defaults.yaml:9]: Seems to work with the ".", i'm using the current form of it.

File: aptcacherng/init.sls (hunk 2 above)
Comment (jochumdev) [inline aptcacherng/init.sls:25]: Ups forgot to readd on manual merge, will provide another commit in minutes.

**Pull request merged**